### DISCUSSION TITLE
How to create nested menus

### DISCUSSION THREAD

**Discussion by bozosort** [mantinedev/mantine#3336] [Q&A]
I tried doing it this way, but it's not working:

`<Menu width={200} >
                <Menu.Target>
                  <UnstyledButton  >Categories</UnstyledButton>
                </Menu.Target>
                <Menu.Dropdown>
                  <Menu.Item >
                  <Menu width={200} position="right-start" >
                  <Menu.Target>
                  <UnstyledButton>Cat</UnstyledButton>
                </Menu.Target>
                      <Menu.Dropdown>
                        <Menu.Item>Cat A1</Menu.Item>
                        <Menu.Item>Cat A2</Menu.Item>
                      </Menu.Dropdown>
                   </Menu>
                  </Menu.Item>
                  <Menu.Item >Cat B</Menu.Item>
                </Menu.Dropdown>
          </Menu>`

**Comment by rtivital** [comment]
Nested menus are not currently supported

**Comment by davigmacode** [comment]
It currently works with `closeOnItemClick={false}`, but still has odd behavior

![Nested Menu](https://bafybeiewztxd3tmvmzxsqoy7moo66kqqengnk5tzzrnewjq7eqvzl4hlv4.ipfs.nftstorage.link/)

```
<Menu width={200} shadow="md" position="bottom-start" closeOnItemClick={false}>
  <Menu.Target>
    <Button>Toggle menu</Button>
  </Menu.Target>

  <Menu.Dropdown>
    <Menu.Item>Menu Level 1</Menu.Item>

    <Menu width={200} shadow="md" position="right-start" closeOnItemClick={false}>
      <Menu.Target>
        <Menu.Item rightSection={<IconChevronRight size={18}/>}>
          Menu Level 1
        </Menu.Item>
      </Menu.Target>

      <Menu.Dropdown>
        <Menu.Item>Menu Level 2</Menu.Item>
        <Menu.Item>Menu Level 2</Menu.Item>
        <Menu width={200} shadow="md" position="right-start" closeOnItemClick={false}>
          <Menu.Target>
            <Menu.Item rightSection={<IconChevronRight size={18}/>}>
              Menu Level 2
            </Menu.Item>
          </Menu.Target>

          <Menu.Dropdown>
            <Menu.Item>Menu Level 3</Menu.Item>
            <Menu.Item>Menu Level 3</Menu.Item>
          </Menu.Dropdown>
        </Menu>
      </Menu.Dropdown>
    </Menu>
  </Menu.Dropdown>
</Menu>
```
